database/migrations/2019_07_17_000001_make_client_secret_nullable.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class MakeClientSecretNullable extends Migration
```

Comment:
The migration isn't 100% necessary as we could just use the empty string for public clients but it seems more explicit to use `NULL`.